### DISCUSSION TITLE
My relationship with the Authentication app has been cut off

### DISCUSSION THREAD

**Discussion by latifaltay** [community/community#146024] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I can log in to github with mobile approval and recovery codes, but I cannot remove the Authentication app configuration from my github account. Can you help?

**Comment by V2U-Services** [comment]
The first thing to do if you can not access an authentication app is to see if you saved backup codes during setup. These codes can assist you in getting back in. Otherwise, get in touch with the platform you are locked out of and use their recovery procedure to confirm your identity. The next time, think about safely storing backups!

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @latifaltay ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by tyagraj777** [comment]
Good question, here are some steps which can help out ....

If you are unable to remove the Authentication app configuration from your GitHub account but can log in with mobile approval and recovery codes, follow these steps to regain full control and manage your two-factor authentication (2FA) settings:

**Steps to Remove the Authentication App Configuration:**
1. Log In with Recovery Codes:

- [ ] Log in to your GitHub account using your recovery codes.
- [ ] Recovery codes bypass the 2FA authentication app requirement

.

2. Navigate to Security Settings:

- [ ] After logging in, go to your GitHub account settings by clicking your profile picture in the top-right corner and selecting "Settings".
- [ ] In the left-hand menu, navigate to "Account security".

3. Disable Two-Factor Authentication:

- [ ] Under the "Two-factor authentication" section, click "Disable".
- [ ] You may need to enter your password to confirm this action.

4. Reconfigure Two-Factor Authentication (Optional):

- [ ] If you want to set up 2FA again, click "Enable two-factor authentication" and follow the instructions to link your account to a new authentication app or generate new recovery codes.

Keep in mind ...!!!

- Recovery codes are one-time-use. Generate a new set after reconfiguring your 2FA settings.
- Ensure your email address on GitHub is updated and verified for account recovery support.

hope this helps !!
